projects/ServoControl/config/UOStateMachine.cpp

```diff
@@ -104,39 +116,83 @@ STATE_DEFINE(UOStateMachine, Control, UOSMData)
         // Check if enable bit is set
         bool enabled = eventNbr > 0 && (eventNbr & EVENT_ENABLE_MASK);
 
-        #if USE_OUT1 == 1
+        #if USE_SV01 == 1
+            if (enabled)
+            {
+                // Open SV01 is the first bit is set
+                bool open = (eventNbr & SV01_EVENT_ENABLE_MASK) > 0;
+
+                gpioData.digitalOutputMap.insert({SV01_NAME, open ? SV01_OPEN : SV01_CLOSE});
+                
+                if(open) {
+                    SPDLOG_LOGGER_INFO(logger, "ServoControlSM::Control SV01 OPEN");
+                } else {
+                    SPDLOG_LOGGER_INFO(logger, "ServoControlSM::Control SV01 CLOSE");
+                }    
+            }
+        #endif
+
+        #if USE_SV02 == 1
+            if (enabled)
+            {
+                // Open SV02 if the second bit is set
+                bool open = (eventNbr & SV02_EVENT_ENABLE_MASK) > 0;
+
+                gpioData.digitalOutputMap.insert({SV02_NAME, open ? SV02_OPEN : SV02_CLOSE});
+
+                if(open) {
+                    SPDLOG_LOGGER_INFO(logger, "ServoControlSM::Control SV02 OPEN");
+                } else {
+                    SPDLOG_LOGGER_INFO(logger, "ServoControlSM::Control SV02 CLOSE");
+                } 
+            }
+        #endif
+
+        #if USE_PWM_SBV01 == 1
             if (enabled)
             {
-                // Open OUT1 is the first bit is set
-                bool open = eventNbr & OUT1_EVENT_ENABLE_MASK;
+                // Open SBV01 if the third bit is set
+                bool open = (eventNbr & SBV01_EVENT_ENABLE_MASK) > 0;
 
-                gpioData.digitalOutputMap.insert({OUT1_NAME, open ? OUT1_OPEN : OUT1_CLOSE});
+                gpioData.pwmOutputMap.insert({SBV01_NAME, open ? SBV01_OPEN : SBV01_CLOSE});
 
-                std::cout << "ServoControlSM::Control OUT1 " << (open ? "OPEN" : "CLOSE") << "\n";
+                if(open) {
+                    SPDLOG_LOGGER_INFO(logger, "ServoControlSM::Control SBV01 OPEN");
+                } else {
+                    SPDLOG_LOGGER_INFO(logger, "ServoControlSM::Control SBV01 CLOSE");
+                } 
             }
         #endif
 
-        #if USE_PWM1 == 1
+        #if USE_PWM_SBV02 == 1
             if (enabled)
             {
-                // Open PWM2 if the second bit is set
-                bool open = eventNbr & PWM1_EVENT_ENABLE_MASK;
+                // Open SBV02 if the third bit is set
```

Comment:
Sorry, I know this is nitpicky. I think we could maybe even remove those comments. Removing those comments would cause fewer issues of the comment and the code not be in sync.
```suggestion
                // Open SBV02 if the fourth bit is set
```